index.html

```diff
@@ -171,7 +171,7 @@ <h2>About BIDS</h2>
         <p>A good introduction to the BIDS standard can be found in the <a href="http://www.nature.com/articles/sdata201644">paper published in Nature Scientific Data</a>.</p>
```

Comment:
This is to align the format of this paragraph with the following ones (imperative mode instead of passive mode)

```suggestion
        <p>Find a good introduction to the BIDS standard in the <a href="http://www.nature.com/articles/sdata201644">paper published in Nature Scientific Data</a>.</p>
```